opentech/apply/dashboard/templates/dashboard/reviewer_dashboard.html

```diff
@@ -12,31 +16,64 @@ <h1 class="gamma heading heading--no-margin heading--bold">Dashboard</h1>
         {% endblock %}
     </div>
 </div>
+
 <div class="wrapper wrapper--large wrapper--inner-space-medium">
-        <div class="wrapper wrapper--large wrapper--inner-space-medium">
-            <h3>Applications to review</h3>
-            {% if my_review.data %}
-                <p>These are applications for which your review has been requested.</p>
-                {% render_table my_review %}
-            {% else %}
-                No reviews to complete
+    <div class="wrapper wrapper--large wrapper--inner-space-medium">
+
+        <h4 class="heading heading--normal">
+            Waiting for your review <span class="heading heading--submission-count">{{ in_review_count }}</span>
+        </h4>
+
+        {% if my_review.data %}
+            {% render_table my_review %}
+
+            {% if display_more %}
+                <div class="all-submissions-table__more">
+                    <a href="{% url 'apply:submissions:list' %}">Show all</a>
+                </div>
             {% endif %}
-        </div>
+
+        {% else %}
+            <div class="grid grid--seventy-thirty wrapper wrapper--large ">
```

Comment:
I dont think we want to put the seventy-thirty layout in just yet. we will not be adding the `30` component this block of sprints.

Reply:
I'll remove this css class definition entirely.